l5kit/l5kit/cle/metrics.py

```diff
@@ -237,3 +237,42 @@ def compute(self, simulation_output: SimulationOutput) -> torch.Tensor:
         pad_drive_meters = torch.cat((pad, drive_meters))
         driven_miles = pad_drive_meters * self.METER_TO_MILES
         return driven_miles
+
+
+class YawErrorMetric(SupportsMetricCompute):
+    """Yaw error computes the difference between the
+    simulated trajectory yaw and the observed trajectory yaw.
+
+    :param error_function: error function to compute distance
+    """
+    metric_name = "yaw_error"
+
+    def __init__(self, error_function: error_functions.ErrorFunction) -> None:
```

Comment:
Can set the `error_function` here to be the default as the closest angle.

Reply:
I am following the template of `DisplacementErrorMetric`. It doesn't have a default `error_function`. Can I keep as it is currently?

Reply:
For displacement error it makes more sense to have different metrics to compute, but for the yaw the one we're using it is a more natural option, users might confuse that and end up other metrics, so it should be ok to deviate here in this case.

Reply:
Okay, should I still keep the `YawErrorCAMetric` then?